docs/source/quickstart.rst

```diff
@@ -0,0 +1,182 @@
+Quick Start
+===========
+
+The quick start is intended for developers who are ready to dive in to the code, and see an end-to-end example of how they can integrate 🤗 Datasets into their model training workflow. For beginners who are looking for a gentler introduction, we recommend you begin with the :doc:`tutorials <./tutorial>`.
+
+In the quick start, you will walkthrough all the steps to fine-tune `BERT <https://huggingface.co/bert-base-cased>`_ on a paraphrase classification task. Depending on the specific dataset you use, these steps may vary, but the general steps of how to load a dataset and process it are the same.
```

Comment:
Since this quick start is quite similar to the Hugging Face [course](https://huggingface.co/course/chapter3?fw=pt), would it make sense to point readers there for more information if they want it?